packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -299,7 +307,7 @@ function startServerInNewWindow(port, terminal, reactNativePath) {
    */
   const scriptsDir = path.dirname(launchPackagerScript);
   const packagerEnvFile = path.join(scriptsDir, packagerEnvFilename);
-  const procConfig: Object = {cwd: scriptsDir};
+  const procConfig: any = {cwd: scriptsDir};
```

Comment:
do we need `any`?

Reply:
Changed type annotation here to more strict.